package.json

```diff
@@ -313,7 +313,7 @@
     "grunt-npm": "0.0.2",
     "jasmine-core": "^2.3.4",
     "json3": "^3.3.2",
-    "karma-browserify": "^5.0.1",
+    "karma-browserify": "^5.0.2",
```

Comment:
Ok, but travis will be failing either way because it doesn't work with the current version of browserify